doc/api/packages.md

```diff
@@ -660,6 +660,9 @@ is provided below to assist with ecosystem coordination.
 
 * `"types"` - can be used by typing systems to resolve the typing file for
   the given export. _This condition should always be included first._
+* `"source"` - can be used to reference unminified / unoptimized source versions
+  to use for debugging workflows, that will behave identically to any non-source
+  variants.
```

Comment:
```suggestion
* `"source"` - can be used to reference unminified, unoptimized, _runnable_
  source JavaScript files. These can be used for debugging workflows, that will
  behave identically to any other variants.
```

Reply:
The only other word I can think of off-hand that captures the same meaning might be `"unoptimized"`.

Reply:
`"pretty"` is close, but it's not just unminification, it's also having separated source files structured by the author.